application-tracking.md

```diff
@@ -0,0 +1,49 @@
+# Application Tracking
+
+## Principles
+
+### Why?
+
+We should not be asking why we should be tracking an event in an application, we should be tracking events by default. Events containing sensitive information such as user or card details should still be tracked but with the sensitive data removed or masked.
+
+### What?
+
+Applications should have tracking enabled for all features that a user can interact with. When new features are written adding tracking should be a requirement for the definition of done.
```

Comment:
it is possible to also track what a customer sees but doesn't interact with, can be handy for marketing banners etc